rust/core-lib/src/find.rs

```diff
@@ -78,6 +78,10 @@ impl Find {
         self.hls_dirty
     }
 
+    pub fn case_matching(&self) -> bool {
+        self.case_matching == CaseMatching::Exact
```

Comment:
Should this support a "smart" mode like for example ripgrep does?

Reply:
There's a reasonable case that `SelectionForFind` should always be case sensitive, although this would be different from the system behaviour on macOS at least, and different from the behaviour of ST3.

(which is to say that I'm not sure?)

Reply:
I took Sublime as reference for this but I think it might be best to add an option `case_sensitive` to the RPC command.